samples/benchmarks/rna-named.yml

```diff
@@ -0,0 +1,14 @@
+- workflow_id: d6d3c2119c4849e4
+  output_history_base_name: RNA-seq
+  reference_data:
+    - name: Reference Transcript (FASTA)
+      dataset_id: 50a269b7a99356aa
+  runs:
+    - history_name: 1
+      inputs:
+      - name: FASTQ RNA Dataset
+        dataset_id: 28fa757e56346a34
```

Comment:
should dataset_ids correspond with datasets.yml in the samples folder?

Reply:
No.  The `dataset_id` in the benchmark file is the ID (or name) of the dataset in the Galaxy history.  So something like `ucsc.hg38.knownGene.gtf`.  That sample should likely be updated or removed. The `datasets.yml` (and `histories.yml` and`workflows.yaml`) files are used to provide aliases to URLs that are used when importing datasets/histories/workflows into Galaxy.

```
abm main dataset import rna-1g
abm main workflow import rnaseq-pe
```